src/test/features/ccj/routes/ccjGuard.ts

```diff
@@ -51,6 +53,58 @@ describe('CCJ guard', () => {
           .set('Cookie', `${cookieName}=ABC`)
           .expect(res => expect(res).to.be.successful.withText('County Court Judgment requested'))
       })
+
+      it('should NOT redirect to dashboard when claim not eligible for CCJ on re determination page', async () => {
+        const route: string = Paths.redeterminationPage.evaluateUri({ externalId: 'b17af4d2-273f-4999-9895-bce382fa24c8' })
+        claimStoreServiceMock.resolveRetrieveClaimByExternalId({
+          respondedAt: MomentFactory.currentDateTime(),
```

Comment:
Please extract this JSON object and reuse in tests

Reply:
done